### ISSUE TITLE
Hover to display multiple formats?

### ISSUE THREAD

**Issue by mattmackay76** [HaaLeo/vscode-timing#30]
**Is your feature request related to a problem? Please describe.**

I get logs from server in UTC Epoch (milliseconds format, I'd like to know the YYYY/MM/DD hh:ss  in both utc0 and local (utc7 for me) at the same time. I tried playing with the "timing.hoverTimestamp.targetFormat": "YYYY-MM-DD HH:mm:ss" but couldn't figure out how to get what I'm looking for. LLLL is local but I have no control over what it looks like.

**Comment by HaaLeo** [comment]
@mattmackay76 thank you for your contribution. If I get you right, you got two requests: 
1. Enable the hover preview to display multiple formats and
2. For a custom format in the hover preview, enable the user to indicate whether the custom format should be localized or not. 

# Possible Approach
A possible approach could be to change the `"timing.hoverTimestamp.targetFormat"` setting to allow either a _string_ or an _array_ of objects. For example: 
```json
{
    "timing.hoverTimestamp.targetFormat": [
        {
            "format": "YYYY/MM/DD hh:ss",
            "localize": true
        }
     ]
}
```

However I do not know whether the [momentjs](https://momentjs.com/docs) supports localization of custom formats.   
Further I am busy the next two weeks. So if anybody wants to take a look at this earlier, go ahead 🚀 🙂.

**Comment by mattmackay76** [comment]
Thanks for building this, it's pretty awesome and thank you for responding!

I actually attempted to put an array in there for hoverTimestamp.targetFormat like ["local","utc","yyyy-MM-dd"] but alas it didn't work :(... but yes, your suggestion is exactly what I was hoping for. 

I wouldn't even know where to start to try and work on this myself. I have no vscode extension experience but I'll take a look. 

Thanks again

**Comment by HaaLeo** [comment]
Released in v2.1.0
